lib/seam/base_resource.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require_relative "utils/deep_hash_accessor"
```

Comment:
```suggestion
require_relative "./deep_hash_accessor"
```

Reply:
2cec68f6573cb4902aa1069f5e14142cbb75c1cb

Reply:
Should we move `lib/seam/utils/action_attempt_utils.rb` out of utils as well?